README.md

```diff
@@ -21,13 +21,13 @@ This is the fastest way to get started.
 See the [installation documentation](https://bazelbuild.github.io/rules_nodejs/install.html) for details and alternative methods, or if you already have a Bazel project and you're adding Node/JavaScript support to it.
 
 ```sh
-$ npm init @bazel
+$ npm init @bazel myproject   # creates './myproject/', sets up Bazel workspace
 ```
 
 or if you prefer yarn,
 
 ```sh
-$ yarn create @bazel
+$ yarn create @bazel myproject
 ```
 
 > These commands are equivalent to `npx @bazel/create` which downloads the latest version of the `@bazel/create` package from npm and runs the program contained.
```

Comment:
could you update line 35, it now needs to ask you to run with `--help` to see the other flags like typescript

Reply:
`npm init @bazel --help` and `yarn create @bazel --help` don't appear to work, so the best I can think of is that I suggest running `npm init @bazel` or `yarn create @bazel`